Examples/LocationManager/LocationManager/LocalSearchClient/LocalSearchClient.swift

```diff
@@ -0,0 +1,10 @@
+import ComposableArchitecture
+import MapKit
+
+struct LocalSearchClient {
+  var search: (MKLocalSearch.Request) -> Effect<LocalSearchResponse, Error>
```

Comment:
`LocalSearchClient` represents a TCA-friendly wrapper around the `MKLocalSearch` API.